examples/getstarted/src/index.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
+  /**
+   * An asynchronous register function that runs before
+   * your application is loaded.
```

Comment:
All the files are already loaded but the internal service haven't been initialized yet.

```suggestion
   * your application is initialized.
```